lib/rex/post/meterpreter/packet_dispatcher.rb

```diff
@@ -66,9 +66,12 @@ def initialize_passive_dispatcher
     self.waiters    = []
     self.alive      = true
 
+    # Ensure that there is only one leading and trailing slash on the URI
+    resource_uri = "/" + self.conn_id.to_s.gsub(/(^\/|\/$)/, '') + "/"
```

Comment:
We also use Base64Url (vs Base64), which transcribes slashes and and plus signs to - and _ instead.